content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -135,8 +145,18 @@ To create filters or `group by` fields for these tags, you must first create fac
 {{% tab "Cucumber" %}}
 Set the `NODE_OPTIONS` environment variable to `-r dd-trace/ci/init`. Run your tests as you normally would, specifying the environment where the tests are run in the `DD_ENV` environment variable. For example, set `DD_ENV` to `local` when running tests on a developer workstation, or `ci` when running them on a CI provider:
 
-{{< code-block lang="bash" >}}
+```bash
 NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app yarn test
+```
+
+**Important**: if you have set a value for `NODE_OPTIONS`, make sure it does not overwrite `-r dd-trace/ci/init`. This can be done via the `${NODE_OPTIONS:-}` clause:
```

Comment:
```suggestion
**Important**: if you set a value for `NODE_OPTIONS`, make sure it does not overwrite `-r dd-trace/ci/init`. This can be done using the `${NODE_OPTIONS:-}` clause:
```